packages/website/pages/tokens.js

```diff
@@ -203,6 +203,14 @@ export default function Tokens({ isLoggedIn }) {
                 </div>
               </div>
             </When>
+            <When condition={tokens.length === 1}>
+              <div className="flex flex-col items-center">
+                <p className="font-black mt-10">
+                  Want to test the token quickly? Upload a file in 
+                  {" "}<a className="underline" href={`https://Upload-with-Token-Node-demo.web3storage.repl.co/?token=${tokens[0].secret}`} target="_blank" rel="noreferrer">repl.it</a>
```

Comment:
nit: the url gets redirected to the lowercase equivalent, so let's just lowercase it here. 

Also let's not pass in the first token to the repl. The user should copy the token they want and paste it into the repl window manually. If i have 3 tokens listed, i'd like to know which token i am testing.

Also also, passing the token to repl.co in as a search param will have it appear in repl.co's http request logs which is bad, and the users brower history, which is less bad, but still undesirable.

Reply:
Alright, I'll also change it so that the demo can show with any number of tokens, right now it's only if you have 1 token only.